test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -853,6 +874,33 @@ redis_query(Node, Query) ->
     RedisWorker = rpc(Node, wpool_pool, best_worker, [mod_global_distrib_mapping_redis]),
     rpc(Node, eredis, q, [RedisWorker, Query]).
 
+eventually_true(Description, _Predicate, Interval, 0) ->
```

Comment:
It would be better to reuse `wait_until`, only perhaps extend it with `Description`.